includes/Settings/SettingsDebug.php

```diff
@@ -1007,7 +1012,12 @@ public function get_server_config(): array {
 			),
 			'WP Memory Limit' => array(
 				/* translators: <a> tags */
-				'required' => sprintf( __( 'Recommended: 128MB (more for plugin-heavy setups<br/>See: %1$sIncreasing the WordPress Memory Limit%2$s', 'woocommerce-pdf-invoices-packing-slips' ), '<a href="https://docs.woocommerce.com/document/increasing-the-wordpress-memory-limit/" target="_blank">', '</a>' ),
+				'required' => __( 'Recommended: 128MB (more for plugins with heavy setups)', 'woocommerce-pdf-invoices-packing-slips' ) . '<br/>' . sprintf(
```

Comment:
This sentence doesn't seem quite right. How about this instead?

> Recommended: 128MB (especially for plugin-heavy setups)

Reply:
Good suggestion.
I actually wanted to change it to `(more for websites with heavy setups)` but I for missed replacing plugins with websites.